Sources/LayoutSpecBuilders/Layout/CornerLayout.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CornerLayout.swift
+//  TextureSwiftSupport
+//
+//  Created by arya.cia on 14/03/22.
+//  Copyright © 2022 muukii. All rights reserved.
+//
+
+import Foundation
+
+public struct CornerLayout<CornerContent, Content> : _ASLayoutElementType where CornerContent : _ASLayoutElementType, Content : _ASLayoutElementType {
+
+  public let content: Content
+  public let cornerContent: CornerContent
+  public let location: ASCornerLayoutLocation
+  
+  public init(
+    location: ASCornerLayoutLocation = .topRight,
```

Comment:
Is this default parameter according to Texture's pure component ASCornerLayoutSpec?

Reply:
in the texture's code itself, they do not specify the default location, but the most common case I found is to set the corner at the top-right of the node, even the texture docs' example shows the top-right location, should I delete the default value? wdyt

Reply:
It original implementation does not have a default value, it should be removed.
Because I think basically TextureSwiftSupport should be a basic wrapper for Texture.

Reply:
removed here as well https://github.com/TextureCommunity/TextureSwiftSupport/pull/82/commits/462250732baca44a737bc73513612396ed7f1c99